api/src/main/java/com/community/api/endpoint/serviceProvider/ServiceProviderEntity.java

```diff
@@ -2,12 +2,16 @@
 
 
 import com.community.api.entity.*;
+import com.community.api.utils.Document;
+import com.community.api.utils.ServiceProviderDocument;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
 
 import javax.persistence.*;
 import javax.validation.constraints.*;
```

Comment:
Again avoid importing all the classes just import the respective class you need.
![image](https://github.com/user-attachments/assets/c1d0a966-5a68-4309-8ff9-411e4df099b2)